CHANGELOG.md

```diff
@@ -1,3 +1,35 @@
+## 1.1.0 / 2016-09-01
+
+* [CHANGE] Kubernetes SD: Drop `__meta_kubernetes_role`.
+* [FEATURE] Add `quantile` and `quantile_over_time`.
+* [FEATURE] Add `stddev_over_time` and `stdvar_over_time`.
+* [FEATURE] Add various time and date functions.
+* [FEATURE] Added `toUpper` and `toLower` formatting to templates.
+* [FEATURE] Allow relabeling of alerts.
+* [FEATURE] Allow URLs in targets defined via a JSON file.
+* [FEATURE] Add idelta function.
+* [FEATURE] 'Remove graph' button on the /graph page.
+* [FEATURE] Kubernetes SD: Add node name and host IP to pod discovery.
+* [FEATURE] New remote storage write path. EXPERIMENTAL!
+* [ENHANCEMENT] Improve time-series index lookups.
+* [ENHANCEMENT] Forbid invalid relabel configurations.
+* [ENHANCEMENT] Improved various tests.
+* [ENHANCEMENT] Add crash recovery metric 'started_dirty'.
+* [ENHANCEMENT] Fix (and simplify) populating series iterators.
+* [ENHANCEMENT] Add job link on target page.
+* [ENHANCEMENT] Allow query parameters for the /graph endpoint.
```

Comment:
This will break Grafana and Promdash "Open this graph in Prometheus" integrations as we don't provide a redirect from the old format. I don't think this needs to block the release, but something to watch out for.